tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -45,19 +45,39 @@ Create VCH Within Datacenter
     Post Path Under Target    datacenter/${dcID}/vch    ${data}
 
 
-Inspect VCH ${name}
-    ${RC}    ${OUTPUT}=    Run And Return Rc And Output    bin/vic-machine-linux inspect config --target=%{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --name=${name} --format raw
+Inspect VCH Config ${name}
+    ${RC}    ${OUTPUT}=    Run And Return Rc And Output    bin/vic-machine-linux inspect config --target=%{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource=%{TEST_RESOURCE} --name=${name} --format raw
     Should Be Equal As Integers    ${RC}    0
     Set Test Variable    ${OUTPUT}
 
 
+Inspect VCH
+    [Arguments]  ${name}  ${expected}
+    ${rc}    ${output}=    Run And Return Rc And Output  bin/vic-machine-linux inspect --name=${name} --target=%{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource=%{TEST_RESOURCE}
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  ${expected}
+    [Return]  ${output}
+
+
 Get VCH ${name}
     Get Path Under Target    vch
     ${id}=    Run    echo '${OUTPUT}' | jq -r '.vchs[] | select(.name=="${name}").id'
 
     Get Path Under Target    vch/${id}
 
 
+Get VCH Docker Info
```

Comment:
These should really live in the VCH-Util library, and unified with the unparameterized versions (we can use the default values mechanism to preserve existing behaviour). Thanks for the modification!

Reply:
Yes I think they should be moved too. I can move both the `Get VCH Docker Info` and `Inspect VCH` to util library, so they can co-exist with the unparameterized ones to preserve existing behavior.

Having two keywords of essentially the same functionality seems to be the only way to go for now. Re-using the keywords from util library has been hard because:
1. In the API create robot test, the `VCH-NAME` test env var is not reset for every test, so we can't use the unparameterized (using env vars) versions from util library 
2. The `VCH Docker Info` provided in util library calls inspect without specifying `compute-resource`, and it fails in our case with VCH created by API.

Reply:
Should be able to just default to %{TEST_RESOURCE}

Reply:
Do we want to run a full CI to check or we can just assume that existing behaviors are preserved